java/gazelle/testdata/java_export_targets/module1/src/main/java/com/example/module1/foo/BUILD.out

```diff
@@ -0,0 +1,7 @@
+load("@rules_java//java:defs.bzl", "java_library")
+
+java_library(
+    name = "foo",
+    srcs = ["Module1Foo.java"],
+    visibility = ["//:__subpackages__"],
```

Comment:
This will need to be constrained to prevent people accessing the target by accident.

Reply:
done now.